src/server/scripts/Commands/cs_reload.cpp

```diff
@@ -931,6 +931,7 @@ class reload_commandscript : public CommandScript
         sLog->outString("Re-Loading Page Texts...");
         sObjectMgr->LoadPageTexts();
         handler->SendGlobalGMSysMessage("DB table `page_texts` reloaded.");
+        handler->SendGlobalGMSysMessage("You will need to delete your client cache to see the changes.");
```

Comment:
why not put it on the same line? seems more fitting to me because when u reload all tables, you will have tons of lines

Reply:
To have it in a new line

Reply:
or we do this instead? 
```cpp
handler->SendNotification("You will need to delete your client cache to see the changes.");
```
?